src/librustdoc/html/layout.rs

```diff
@@ -53,6 +55,7 @@ pub fn render<T: Print, S: Print>(
     <link rel=\"stylesheet\" type=\"text/css\" href=\"{static_root_path}rustdoc{suffix}.css\" \
           id=\"mainThemeStyle\">\
     {style_files}\
+    <script id=\"default-settings\"{default_settings}></script>\
```

Comment:
As a side-note: this ID should be added to the list of IDs that can't be used in the HTML code (look for any other ID, look for `mainThemeStyle` and you should find where you have to add it).